### DISCUSSION TITLE
Best practices for analysis_options

### DISCUSSION THREAD

**Discussion by AdrienLemaire** [rrousselGit/riverpod#3507] [Q&A]
Hi,

I've discovered the riverpod_lint package, which I'd like to use on top of my existing options (very_good_analysis using the dart_code_metrics plugin), and am realizing I cannot have more than 1 plugin listed in `analyzer.plugins`.

Moreover, when using only the custom_lint, `flutter lint` reports no issues found, when dart run custom_lint finds 4.

I wouldn't mind having to run both flutter analysis (for very_good_analysis) and custom_lint (for riverpod_lint), but they are conflicting by both using the `analysis_options.yaml` file.

How do you get around this ? How can I get the best of both worlds without having to rewrite all the linter rules from both plugins myself, at the risk of missing new ones when these plugins add some ?

**Comment by rrousselGit** [comment]
What do you mean by conflict between very_good_analysis and riverpod_lint?
very_good_analysis is not an analyzer_plugin. You can use both very_good_analysis and riverpod_lint without issue.
